packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Cover.Example.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
-import './CalloutExample.scss';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
 import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { getId, classNamesFunction, css } from 'office-ui-fabric-react/lib/Utilities';
+import { calloutExampleStyle, ICalloutExampleStyle } from './Callout.Examples.Styles';
```

Comment:
Putting the example styles in their own file still prevents export from working, even if it's a TS file rather than SCSS. Also, we typically recommend using `mergeStyleSets` for examples (rather than `classNamesFunction`) for simplicity. (Same comment applies to other examples.)

Reply:
Got it. I think fixed it (following the example in the Basic file)